l10n/en/firefox/browsers/best-browser.ftl

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org//firefox/browsers/best-browser/
+
+best-browser-find-your-best-browser = Find your best browser for speed, privacy and security.
+best-browser-so-many-browser-options = So many browser options, but there’s only one that works best for your needs. The best browser for you should offer both speed and privacy protection.
+best-browser-privacy-speed-and-security = Privacy, speed, and security.
+best-browser-how-to-choose-the-best = How to choose the best browser for you.
+best-browser-the-internet-has-become = The internet has become as essential as electricity and running water, so choosing the best browser for you is more important than ever. The internet is a second office, a teacher and sometimes a medical advisor, even if your actual doctor would prefer you didn’t look up your symptoms online.
+best-browser-in-the-mid-nineties = In the mid-nineties, { -brand-name-netscape }, { -brand-name-ie } and AOL dominated the landscape. It was a simpler time when the sweet melody of dial-up internet rang across the land. You learned the meaning of patience waiting for web pages to load. Back then, all that mattered was browser speed.
```

Comment:
Should we add AOL to `brands.ftl`? `{ -brand-name-aol }`

Reply:
We touched on AOL briefly in the L10n review, and decided not to add AOL for now since it's so far only mentioned in this page: https://github.com/mozilla-l10n/www-l10n/pull/149#discussion_r535480501